package.json

```diff
@@ -163,7 +163,8 @@
             "@typescript-eslint/explicit-function-return-type": "off",
             "@typescript-eslint/no-unused-vars": "warn",
             "@typescript-eslint/camelcase": "off",
-            "@typescript-eslint/no-explicit-any": "off"
```

Comment:
@JustinaMary Disabled this rule se we can type the updates parameter in the more generic update() method